modules/sandboxed-containers-deploying-workloads-with-kata-runtime-web.adoc

```diff
@@ -29,48 +29,29 @@ To deploy a pod-templated workload in a sandboxed container, you must manually a
 .Example for Pod
```

Comment:
Same nit as mentioned in the `main` PR: https://github.com/openshift/openshift-docs/pull/64266#pullrequestreview-1611413746